### PR TITLE
VBID-5082 stack trace when restarting trafficking ui

### PR DESCRIPTION
**Pull request by robolange**
Fix race condition in which watch & serve were being called concurrently with clean (as part of build).

### CODE REVIEW

**Comment by chrismcbride** [comment]
+1

**Comment by jawsthegame** [comment]
:+1: :+1:

File: src/index.coffee [hunk 1]
```diff
@@ -91,7 +91,12 @@ module.exports = (projectConfig={}) ->
     runSequence('clean', ['src', 'static', 'style'])
 
 
-  gulp.task 'watch:serve', ['watch', 'serve']
+  gulp.task 'serve', ->
+    runSequence('clean', ['src', 'static', 'style'], ['_serve'])
```
Comment (cliscum) [inline src/index.coffee:95]: Can this be `runSequence('build', '_serve')`?

File: src/index.coffee [hunk 2]
```diff
@@ -91,7 +91,12 @@ module.exports = (projectConfig={}) ->
...
+
+
+  gulp.task 'watch:serve', ->
+    runSequence('clean', ['src', 'static', 'style'], ['watch', '_serve'])
```
Comment (cliscum) [inline src/index.coffee:99]: And here? `runSequence('build', ['watch', '_serve'])`

File: src/index.coffee (hunk 1 above)
Comment (robolange) [inline src/index.coffee:95]: I tried that, but runSequence overall returns asynchronously.

**Pull request merged**